install-librephotos.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+set -exa
+
+# PRE INSTALL
+grep librephotos /etc/group > /dev/null || groupadd -r librephotos
```

Comment:
I 'd have used:
```
id -g librephotos > /dev/null || groupadd -r librephotos
```

same for all groups

Reply:
done
merci